cmd/activator/request_log.go

```diff
@@ -29,7 +30,16 @@ import (
 
 func updateRequestLogFromConfigMap(logger *zap.SugaredLogger, h *pkghttp.RequestLogHandler) func(configMap *corev1.ConfigMap) {
 	return func(configMap *corev1.ConfigMap) {
-		newTemplate := configMap.Data["logging.request-log-template"]
+		var newTemplate string
```

Comment:
again bit of a nit sorry, but could we move this down to just above `if obsconfig.EnableRequestLog`? that way the defaulting of that variable is in one place (and I think it's best if a guard is the first thing if possible anyway)

Reply:
no problem